lang/rs/wireguard/patch/0001-Integrate-CNDP-AF-XDP-with-Rust-Wireguard.patch

```diff
@@ -91,7 +91,7 @@ index 640d8cb..7cb769b 100644
 @@ -5,12 +5,22 @@ license = "MIT"
  name = "wireguard-rs"
  version = "0.1.4"
- 
+
 +[[bin]]
```

Comment:
Changing the patch file could be a problem here, need to verify or we should ignore this file.

Reply:
Yeah, let me reset these and add them to the ignored files... sorry